.kiro/steering/structure.md

```
@@ -0,0 +1,109 @@
+# QuickLog-Solo: プロジェクト構成
+
+## ディレクトリ構造
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)